app/components/Capture.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react';
+import { Button } from 'react-bootstrap';
+import html2canvas from 'html2canvas';
+import ShareScreen from './ShareScreen';
+
+class Capture extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      img: '',
+      isOpen: false,
+    };
+    this.takeScreenshot = this.takeScreenshot.bind(this);
+    this.toggleModal = this.toggleModal.bind(this);
+  }
+
+  toggleModal = () => {
+    this.setState({
+      isOpen: !this.state.isOpen,
+    });
+  };
+
+  takeScreenshot() {
+    const h = this;
+    html2canvas(document.body).then(canvas => {
+      const imgURI = canvas.toDataURL();
+      h.setState({
+        img: imgURI,
+      });
+      this.toggleModal();
+    });
+  }
+
+  render() {
+    const style = {
```

Comment:
don't do inline styles, add them to appropriate scss files or create new if not present.

Reply:
Added scss